engine/access/rest/websockets/connection.go

```diff
@@ -0,0 +1,42 @@
+package websockets
+
+import (
+	"github.com/gorilla/websocket"
+)
+
+// We wrap gorilla's websocket connection with interface
+// to be able to mock it in order to test the types dependent on it
+
+type WebsocketConnection interface {
+	ReadJSON(v interface{}) error
+	WriteJSON(v interface{}) error
+	Close() error
+}
+
+type GorillaWebsocketConnection struct {
```

Comment:
Please add godoc here

Reply:
I feel like it's redundant in this case. The intention is pretty clear. What do you think?

Reply:
I added a comment to this file